MBLD scrambling guidelines (#1155)
thewca/wca-regulations: wca-guidelines.md

- 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
- 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
- 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.

> - 4g+) [RECOMMENDATION] For Multi-Blind Solving, after scrambling all the cubes, the scrambler must verify there are no duplicate scrambles.

Reviewer: it seems a bit odd to list a "must regulation" as a "recommendation". Perhaps "clarification" would be better suited?

Reviewer 2: Indeed, this should not be a recommendation. It will be fixed :)

> - 4g+) [RECOMMENDATION] For Multi-Blind Solving, after scrambling all the cubes, the scrambler should verify there are no duplicate scrambles.

Reviewer 3: ```suggestion
- 4g+) [RECOMMENDATION] For Multi-Blind Solving, after scrambling all the cubes, the scrambler must verify there are no duplicate scrambles.
```

> - 4g+) [RECOMMENDATION] For Multi-Blind Solving, the scrambler should perform an extra check after scrambling to verify there are no duplicate scrambles.

Reviewer 2: ```suggestion
- 4g+) [RECOMMENDATION] For Multi-Blind Solving, after scrambling all the cubes, the scrambler should verify there are no duplicate scrambles.
```

Reviewer 4: LGTM.

> - 4g+) [ADDITION] For Multi-Blind Solving, the scrambler should verify that the puzzles are scrambled correctly and that there are no duplicate scrambles.

Reviewer 4: ```suggestion
- 4g+) [RECOMMENDATION] For Multi-Blind Solving, the scrambler should pay special attention to duplicate scrambles.
```

I think it's better to remove that part, so we don't give the idea that checking the scrambles is a "should". 4g already states that they must do it.
